src/main/groovy/com/blackbaud/templates/project/PermissionsProject.groovy

```diff
@@ -0,0 +1,63 @@
+package com.blackbaud.templates.project
+
+import static com.google.common.base.CaseFormat.LOWER_CAMEL
+import static com.google.common.base.CaseFormat.LOWER_HYPHEN
+
+class PermissionsProject {
+    private BasicProject basicProject
+
+    PermissionsProject(BasicProject basicProject) {
+        this.basicProject = basicProject
+    }
+
+    private String getServiceName() {
+        basicProject.serviceName
+    }
+
+    private String getServicePackage() {
+        basicProject.servicePackage
+    }
+
+    private String getServicePackagePath() {
+        basicProject.servicePackagePath
+    }
+
+    void addPermissions() {
```

Comment:
permissions are solely a rest project concern, consider moving these methods to `RestProject`

Reply:
Moved.